$NetBSD$

Functions returning pointers must be declared before being called.
(The function in question does not appear to be in any header file
and it isn't clear which if any it should be added to.)

--- genmon/genmon.c~	1998-07-15 16:27:49.000000000 +0000
+++ genmon/genmon.c
@@ -56,6 +56,8 @@
 #include <sys/file.h>			/* for access() defines */
 #include <string.h>
 
+EVENT *logstr_to_event(char *s);
+
 #ifndef SLEEPTIME
 # define SLEEPTIME  15			/* after each popen()  */
 #endif
